tests/queries/0_stateless/01800_log_nested.sql

```diff
@@ -0,0 +1,20 @@
+-- TinyTinyLog
```

Comment:
@azat Sorry, what is the point in

TinyTinyLog and StripeStripeLog?

Reply:
If you are asking about typo, that just a typo.
If the question about "what's the point in testing TinyLog and StripeLog" then then answer is "just in case", have you found some issues with this?

Reply:
Yes, I'm asking about duplication, why it is named **Tiny** Tiny Log, is there any meaning in that.
Looks like there isn't. Could you please send a PR that fixes this typo?

Reply:
@CurtizJ You've missed "quite important" issue in review :)

Reply:
>Could you please send a PR that fixes this typo?

Sure, will put it into the queue and submit with other tests fixes.